### ISSUE TITLE
Redeclaration of AppEvents.Name.purchased

### ISSUE THREAD

**Issue by junpluse** [facebook/facebook-swift-sdk#454]
## Checklist

- [x] I've updated to the latest released version of the SDK
- [x] I've searched for existing [GitHub issues](https://github.com/facebook/facebook-swift-sdk/issues)
- [x] I've looked for existing answers on [Stack Overflow](https://facebook.stackoverflow.com) and the [Facebook Developers Group](https://www.facebook.com/groups/fbdevelopers)
- [x] I've read the [Code of Conduct](CODE_OF_CONDUCT.md)
- [x] This issue is not security related and can safely be disclosed publicly on GitHub

## Environment

Describe your dev environment here, giving as many details as possible. If you have them, make sure to include:

- Xcode Version: `10.2.1`
- Swift Version: `5.0.1`
- Installation Platform & Verison: `Cocoapods` version `1.6.1`

## Goals

Remove the declaration of `AppEvents.Name.purchased` from the `FacebookCore` module.

`AppEvents.Name.purchased` is already defined in the `FBSDKCoreKit`.

https://github.com/facebook/facebook-objc-sdk/blob/7d45dd66deb947d7d159443eee49f2baff142ef0/FBSDKCoreKit/FBSDKCoreKit/AppEvents/FBSDKAppEvents.h#L138

But the `FacebookCore` also defines `AppEvents.Name.purchased` as a public extension with the same raw string value.

https://github.com/facebook/facebook-swift-sdk/blob/f02ed1760d9bcee77a4c41a4bb51d4f933fdc2f8/Sources/Core/AppEvents.Name.swift#L25

Why? 🤷‍♂ 

## Expected Results

Use `AppEvents.Name.purchased` with no errors.

## Actual Results

`AppEvents.Name.purchased` causes a compile error: `Ambiguous use of 'purchased'`.

<img width="407" alt="Screen Shot 2019-05-16 at 20 30 06" src="https://user-images.githubusercontent.com/312595/57851207-f1a33a80-781a-11e9-9295-207237054d62.png">

<img width="322" alt="Screen Shot 2019-05-16 at 21 01 50" src="https://user-images.githubusercontent.com/312595/57852334-de459e80-781d-11e9-8778-b48d8ca38867.png">

## Steps to Reproduce

1. Install the SDK 0.7.0 via CocoaPods
2. 👇 

```swift
import FBSDKCoreKit

_ = AppEvents.Name.purchased // Compile error: Ambiguous use of 'purchased'
```

**Comment by choiks14** [comment]
same here

**Comment by PrakashMaharjan** [comment]
Is this issue regarding AppEvent fixed??
I am getting lot of errors when I upgraded to FBSDK 5.0.1
I cannot log the app events like viewed Content.

Currently I am writing below code to log viewed content event:

 let params  = [
            "contentType" : contentType,
            "content" : contentData,
            "contentID" : contentId,
            "currency" : currency
        ]
        
AppEvents.logEvent(AppEvents.Name(rawValue: "viewedContent"), valueToSum: price, parameters: params)
    
Will this code result in App event log in facebook dashboard?


Before FBSDK 5.0.1 in FBSDK 4.44.1, the code was as

let params : AppEvent.ParametersDictionary = [
         .contentType : contentType,
         .content : contentData,
         .contentId : contentId,
         .currency : currency
         ]
         let event = AppEvent(name: .viewedContent, parameters: params, valueToSum: price)
         AppEventsLogger.log(event)

Thanks.

**Comment by siyao1030** [comment]
Same issue here... Please fix as soon as possible.

**Comment by RedFish** [comment]
Same issue 🆙

**Comment by junpluse** [comment]
Fixed in #475 🎉 

This was released as part of [0.8.0](https://github.com/facebook/facebook-swift-sdk/compare/v0.7.0...v0.8.0). Thanks!
